libraries/communication/rtClient/rtdataclient.h

```diff
@@ -136,7 +137,7 @@ class COMMUNICATIONSHARED_EXPORT RtDataClient : public QTcpSocket
 
 private:
     qint32 m_clientID;  /**< Corresponding client id of the data client at mne_rt_server */
-
+    using QTcpSocket::connectToHost;
```

Comment:
why is this needed?

Reply:
why as in why do we all need mne-cpp? jk
please see previous comment.

Reply:
see answer above

Reply:
happier life above.